src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/VirtualizingDataSource.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.TreeViewTests
+{
+	public class CustomContent
+	{
+		public List<CustomContent> Children = new List<CustomContent>();
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)